src/store/mutations.ts

```diff
@@ -40,14 +41,24 @@ export default {
 	setSubclasses( state: RootState, payload: { subclasses: boolean; conditionIndex: number } ): void {
 		state.conditionRows[ payload.conditionIndex ].subclasses = payload.subclasses;
 	},
+	setConditionRelation( state: RootState, payload: { value: ConditionRelation; conditionIndex: number } ): void {
+		state.conditionRows[ payload.conditionIndex ].conditionRelation = payload.value;
+	},
 	setErrors( state: RootState, errors: Error[] ): void {
 		state.errors = errors;
 	},
 	addCondition( state: RootState ): void {
-		state.conditionRows.push( getFreshConditionRow() );
+		if ( state.conditionRows.length === 0 ) {
```

Comment:
This can be easily summarized:
`state.conditionRows.push( getFreshConditionRow( state.conditionRows.length === 0 ) );`

Reply:
wait what???

Reply:
I think that is not very easy to read, IMHO

Reply:
On second thoughts.. yes, it looks better.